Mathlib/Algebra/Module/Injective.lean

```diff
@@ -3,11 +3,10 @@ Copyright (c) 2022 Jujian Zhang. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
```

Comment:
You might add an `assert_not_exists` to this file

Reply:
Good idea, done.